src/Mvc/Mvc.Razor/src/Infrastructure/DefaultFileVersionProvider.cs

```diff
@@ -97,7 +98,7 @@ public string AddFileVersionToPath(PathString requestPathBase, string path)
 
         private static string GetHashForFile(IFileInfo fileInfo)
         {
-            using (var sha256 = CryptographyAlgorithms.CreateSHA256())
+            using (var sha256 = SHA256.Create())
```

Comment:
Why isn't this one using the static method?

Reply:
It's hashing a stream and the one-shot doesn't have a stream overload, only `byte[]` and `ReadOnlySpan<byte>`.